.github/workflows/tsc.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: ['next', 'beta', 'latest', '~4.7']
+        target: ['next', 'beta', 'latest', '~4.8', '~4.7']
```

Comment:
Now that 4.9 has been released, adding 4.8. Whenever a new version is bumped we need to add the previous one to the matrix.